http/1.9/modules/ROOT/pages/http-connector-config-acb.adoc

```diff
@@ -0,0 +1,252 @@
+= Using Anypoint Code Builder to Configure HTTP Connector
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="individual-connector-config"]
+
+=== Configuring Attributes for the HTTP Listener
```

Comment:
```suggestion
== Configuring Attributes for the HTTP Listener
```

Reply:
It's confusing, but we use imperative for tasks within a larger task group - unless this section encompasses more than one task?